Jenkins/PRUnitTestRunner.groovy

```diff
@@ -25,6 +25,23 @@ void prRunner(String cmakeBuildType, String platform, String dockerTag) {
             sh "docker/$platform/runtests.sh"
         }
     }
+//#recipientProviders: [[$class: 'CulpritsRecipientProvider']]
+// maybe remove to
+    post {
```

Comment:
Is this works with scripted pipelines? This seems like declarative pipeline syntax

Reply:
could be that is why it is not working.